src/main/java/uk/gov/hmcts/dm/service/FileContentVerifier.java

```diff
@@ -60,9 +61,11 @@ public boolean verifyContentType(MultipartFile multipartFile) {
                 metadata.add(Metadata.CONTENT_TYPE, multipartFile.getContentType());
             }
             String detected = tika.detect(tikaInputStream, metadata);
-            if (mimeTypeList.stream().noneMatch(m -> m.equalsIgnoreCase(detected))) {
-                log.error(
-                    String.format("Warning. The mime-type of uploaded file is not white-listed: %s", detected));
+
+            if (!StringUtils.endsWithIgnoreCase(detected, PROTECTED)
```

Comment:
why do we need this?

Reply:
This message is accurate in that it indicates that the mime-type is invalid. This is the current message and validity included in this update is StringUtils.endsWithIgnoreCase(detected, PROTECTED). Tika detects that certain file types are protected, but not all of them. Therefore, we must add one extra check in here to pass this case and guarantee that the password validation is completed in PasswordVerifier.java.